fem/ceed/algebraic.cpp

```diff
@@ -0,0 +1,989 @@
+// Copyright (c) 2010-2020, Lawrence Livermore National Security, LLC. Produced
+// at the Lawrence Livermore National Laboratory. All Rights reserved. See files
+// LICENSE and NOTICE for details. LLNL-CODE-806117.
+//
+// This file is part of the MFEM library. For more information and source code
+// availability visit https://mfem.org.
+//
+// MFEM is free software; you can redistribute it and/or modify it under the
+// terms of the BSD-3 license. We welcome feedback and contributions, see file
+// CONTRIBUTING.md for details.
+
+#include "algebraic.hpp"
+
+#include "../bilinearform.hpp"
+#include "../fespace.hpp"
+#include "../ceed/solvers-atpmg.hpp"
+#include "../ceed/full-assembly.hpp"
+#include "../pfespace.hpp"
+#include "../../general/forall.hpp"
+
+namespace mfem
+{
+
+namespace ceed
+{
+
+#ifdef MFEM_USE_CEED
+
+/** Construct a ceed::Operator from a CeedOperator */
+class WrapOperator : public ceed::Operator
```

Comment:
I would rename this class to `OwnedOperator` since this is what it seems to do differently to `Operator`.

Reply:
Actually, can't you just fuse this class with `ceed::Operator`?

Reply:
This class should also be move with `ceed/operator.{hpp,cpp}`.

Reply:
I fused into `ceed::Operator`